### PR TITLE
Bug 2182056: Cloned VM should not use the same PVC of the source VM

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

When cloning a new VM we also need to clone the pvcs, and not use the original ones  

## 🎥 Demo

After:

https://user-images.githubusercontent.com/67270715/228240033-a45dbc2f-f227-4afd-8db4-aac0b63db9b2.mp4

### CODE REVIEW

File: src/views/virtualmachines/actions/components/CloneVMModal/utils/helpers.tsx [hunk 1]
```diff
@@ -68,7 +72,7 @@ export const produceCleanClonedVM = (
     }
 
     delete draftVM?.status;
-    draftVM.spec.dataVolumeTemplates = [];
+    // draftVM.spec.dataVolumeTemplates = [];
```
Comment (metalice) [inline src/views/virtualmachines/actions/components/CloneVMModal/utils/helpers.tsx:75]: Woops

**Comment by pcbailey** [comment]
This looks good to me, but I'd like another reviewer as well since this is really important functionality.

**Comment by avivtur** [comment]
> I assume the fix applys while cloning VM from catalog and instanceType, right?

Yes both worked as expected :)

**Pull request merged**